.github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml

```diff
@@ -144,6 +144,12 @@ jobs:
           echo "token=${{ secrets.SENTRY_AUTH_TOKEN }}" >> .sentryclirc
           cp .sentryclirc api/.sentryclirc
 
+      - name: Set up api environment variables
+        run: |
+          cd api
+          touch .env
+          echo "NODE_ENV=${{ (github.event_name == 'pull_request' && 'staging') || ('production') }}" >> .env
```

Comment:
If this approach works to inject environment variables to the API, it would be more appropriate to have the CI/CD pipeline inject the correct values for `REPAIRS_API_IDENTIFIER` and `REPAIRS_API_BASE_URL` (instead of injecting values for each of them for `staging` and `production` and then determining which to use based on `NODE_ENV`, see [here](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/blob/main/api/gateways/apiRequester.js#L6)).

Reply:
Yes, I agree as long as there isn't a chance of them being exposed in the pipeline log. This is how the output looks at the moment: 

cd api
  touch .env
  echo "NODE_ENV=staging" >> .env

So we would probably need to set masking up to be sure.

Reply:
Let's attempt the above and if it's proves successful, we can instead set the environment variables to the correct values respective of the deployment environment.